packages/fxa-content-server/scripts/test-ci-remote.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -ex
+
+DIR=$(dirname "$0")
+cd "$DIR/.."
+echo -e "\n###################################"
+echo "# testing fxa-content-server"
+echo -e "###################################\n"
+
+function test_suite() {
+  local suite=$1
+  node tests/intern.js \
+    --suites="${suite}" \
+    --fxaAuthRoot=https://api-accounts.stage.mozaws.net/v1 \
+    --fxaContentRoot=https://accounts.stage.mozaws.net/ \
```

Comment:
These URL should be parameterized somewhat similar to the old scripts so that this same test script can be reused against other endpoints. But get stage testing working first.